bulbs/content/tasks.py

```diff
@@ -199,7 +201,9 @@ def post_to_instant_articles_api(content_pk):
                 source,
                 fb_page_id,
                 fb_api_url,
-                fb_token_path)
+                fb_token_path,
+                fb_dev_mode,
```

Comment:
So what exactly do "fb_dev_mode" and "fb_publish" indicate? Maybe replace check with ```if fb_dev_mode or fb_publish``` or similar?